scripts/release-installer

```diff
@@ -20,31 +20,40 @@ verifySupported() {
   fi
 }
 
+download() {
```

Comment:
wouldn't `getReleaseURL` or similar be more accurate here? The actual download happens in `install`

Reply:
changed